dmutils/content_loader.py

```diff
@@ -416,6 +418,27 @@ def get_question(self, framework_slug, question_set, question):
 
         return self._questions[framework_slug][question_set][question].copy()
 
+    def get_message(
+        self, framework_slug, block, framework_status, supplier_status=None
+    ):
+
+        framework = self._content[framework_slug]
+
+        if 'messages' not in framework:
```

Comment:
`ContentBuilder` won't apply. Maybe the messages should be stored in their own `dict`.